documentation/MSBuild-Server.md

```diff
@@ -1,19 +1,26 @@
 # MSBuild Server
 
-MSBuild Server is basically an another type of node which can accept build request from clients and utilize worker nodes in current fashion to build projects. Main purpose of the server node is to avoid expensive MSBuild process start during build from tools like .NET SDK.
+MSBuild Server is basically an another type of node which can accept build request from clients and utilize worker nodes in current fashion to build projects. Main purpose of the server node is to avoid expensive MSBuild process start during build from tools like .NET SDK,
+and preserve caches between builds.
+
+## Usage
+
+The primary ways to use MSBuild are via Visual Studio and via CLI using `dotnet build`/`dotnet msbuild` commands. MSBuild Server is not supported in Visual Studio, becuase Visual studio itself
+works like MSBuild Server. For CLI is the server functionality enabled by default and can be disabled by setting `DOTNET_CLI_DO_NOT_USE_MSBUILD_SERVER` environment variable to value `1`.
```

Comment:
```suggestion
The primary ways to use MSBuild are via Visual Studio and via the CLI using the `dotnet build`/`dotnet msbuild` commands. MSBuild Server is not supported in Visual Studio because Visual studio itself works like MSBuild Server. For the CLI, the server functionality is enabled by default and can be disabled by setting the `DOTNET_CLI_DO_NOT_USE_MSBUILD_SERVER` environment variable to value `1`.
```